package.json

```diff
@@ -6,8 +6,9 @@
   },
   "scripts": {
     "processcss": "node-sass --include-path scss src/styles.scss dist/dist.styles.css",
-    "compilecss": "node scripts/compile-css src/main.js dist/dist.styles.css dist/dist.js",
+    "compilecss": "node scripts/compile-css.js src/main.js dist/dist.styles.css dist/dist.js",
```

Comment:
No need to add the `.js` here and on the file.
Also for your script, it works without it as well :)

Reply:
Yes, but my ide will treat the file as a plain text file. Also i think is better to add the extension to not generate confusion.

Reply:
I see. Ok, this is fine too 👍🏼